### PR TITLE
HW 6

### PR DESCRIPTION
**Pull request by AlinaZvarych**

### CODE REVIEW

File: src/main/java/hw_6/Task1.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package hw_6;
+
+public class Task1 {
+    String name;
```
Comment (AntBelyi) [inline src/main/java/hw_6/Task1.java:4]: access modifier?

File: src/main/java/hw_6/Task1.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    String surname;
```
Comment (AntBelyi) [inline src/main/java/hw_6/Task1.java:5]: access modifier?

File: src/main/java/hw_6/Task1.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
+package hw_6;
+
+public class Task1 {
```
Comment (AntBelyi) [inline src/main/java/hw_6/Task1.java:3]: class name is bad

File: src/main/java/hw_6/Task1.java [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        System.out.println("Surname: " + surname);
+
+        System.out.println("Age: " + age);
+        System.out.println("City: " + city);
+    }
+    public static void printStaticInfo() {
+        System.out.println("Age: " + age);
+        System.out.println("City: " + city);
+    }
+    public static void main(String[] args) {
```
Comment (AntBelyi) [inline src/main/java/hw_6/Task1.java:25]: main method should be move in a separate class?

File: src/main/java/hw_6/Main.java [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+        System.out.println("Довжина: " + array.length());
+
+        // Двічі перевіримо, що немає елементів
+        System.out.println(array.get());
+        System.out.println(array.get());
+
+        System.out.println("Довжина перед тим як покласти елемент: " + array.length());
+        array.put("new Elem");
+        System.out.println("Довжина після того, як поклали: " + array.length());
+        System.out.println(array.get()); // Виводимо перший елемент
```
Comment (AntBelyi) [inline src/main/java/hw_6/Main.java]: there should not be latin words

Comment (AlinaZvarych) [reply src/main/java/hw_6/Main.java]: done

File: src/main/java/hw_6/Task1.java (hunk 1 above)
Comment (AlinaZvarych) [reply src/main/java/hw_6/Task1.java:4]: done

File: src/main/java/hw_6/Task1.java (hunk 2 above)
Comment (AlinaZvarych) [reply src/main/java/hw_6/Task1.java:5]: done

File: src/main/java/hw_6/Task1.java (hunk 3 above)
Comment (AlinaZvarych) [reply src/main/java/hw_6/Task1.java:3]: I just wanted to show that it's task 1 from the hw doc, but ok
done

File: src/main/java/hw_6/Task1.java (hunk 4 above)
Comment (AlinaZvarych) [reply src/main/java/hw_6/Task1.java:25]: done

File: src/main/java/hw_6/Task1.java (hunk 3 above)
Comment (AntBelyi) [reply src/main/java/hw_6/Task1.java:3]: not done

File: src/main/java/hw_6/Task1.java (hunk 2 above)
Comment (AntBelyi) [reply src/main/java/hw_6/Task1.java:5]: not done

File: src/main/java/hw_6/Task1.java (hunk 1 above)
Comment (AntBelyi) [reply src/main/java/hw_6/Task1.java:4]: not done

File: src/main/java/hw_6/Resident.java [hunk 6]
```diff
@@ -0,0 +1,26 @@
+package hw_6;
+
+public class Resident {
+    public String name;
+    public String surname;
+
+    public static int age;
+    public static String city;
```
Comment (AntBelyi) [inline src/main/java/hw_6/Resident.java]: public access modifiers is it a good approach?

File: src/main/java/hw_6/Main.java [hunk 7]
```diff
@@ -0,0 +1,33 @@
+package hw_6;
+import hw_6.MyArray;
```
Comment (AntBelyi) [inline src/main/java/hw_6/Main.java]: please remove unused imports

Comment (AlinaZvarych) [reply src/main/java/hw_6/Main.java]: done

File: src/main/java/hw_6/Resident.java (hunk 6 above)
Comment (AlinaZvarych) [reply src/main/java/hw_6/Resident.java]: changed

File: src/main/java/hw_6/Task1.java (hunk 3 above)
Comment (AlinaZvarych) [reply src/main/java/hw_6/Task1.java:3]: but why? i changed name

File: src/main/java/hw_6/Task1.java (hunk 2 above)
Comment (AlinaZvarych) [reply src/main/java/hw_6/Task1.java:5]: now should be done

File: src/main/java/hw_6/Task1.java (hunk 1 above)
Comment (AlinaZvarych) [reply src/main/java/hw_6/Task1.java:4]: now should be done

Comment (AntBelyi) [reply src/main/java/hw_6/Task1.java:4]: not done

File: src/main/java/hw_6/Task1.java (hunk 2 above)
Comment (AntBelyi) [reply src/main/java/hw_6/Task1.java:5]: not done

File: src/main/java/hw_6/Task1.java (hunk 3 above)
Comment (AntBelyi) [reply src/main/java/hw_6/Task1.java:3]: not done
Screenshot_1

Comment (AlinaZvarych) [reply src/main/java/hw_6/Task1.java:3]: I see.. but I fixed everything 
and when I open 'files changed' section here, I can see what I changed
in my idea i can see the changes as well

**Pull request closed without merging**